en/docs/try-out/account-and-transaction-flow.md

```diff
@@ -86,153 +86,6 @@ Given below is a summary of configurations to follow when deploying the APIs in
 5. Click **Subscribe**.
 
 ## Invoking Account and Transaction API
-
```

Comment:
@line 232, shall we add a note to introduce  CDR Arrangement ID
"The CDR Arrangement ID is a unique string that represents a consent arrangement between a Data Recipient Software Product and Data Holder for a given consumer." and link to https://consumerdatastandardsaustralia.github.io/standards/#identifiers-and-subject-types for more info @silvashenol20